### PR TITLE
Feature/buzzy bee update

### PR DESCRIPTION
**Pull request by ShaneBeee**
This PR adds new bee related items/blocks for the upcoming Minecraft 1.15 Buzzy Bee update

I also refactored some food categories a bit to include sweet berries, and create an `any apple` food category.

### CODE REVIEW

File: foodstuffs.sk [hunk 1]
```diff
@@ -80,17 +84,21 @@ common categories:
...
-	[any] fruit¦s = apple, chorus fruit, melon slice, any golden apple
+	[any] fruit¦s = any apple, chorus fruit, melon slice
 	
 	[any] raw meat¦s = raw beef, raw chicken, raw mutton, raw porkchop, raw rabbit
 	[any] cooked meat¦s = steak, cooked chicken, cooked mutton, cooked porkchop, cooked rabbit
 	[any] meat¦s = any raw meat, any cooked meat
 	[any] monster (food|meat¦s) = rotten flesh, spider eye
 	[any] cooked food¦s = baked potato, any cooked meat, any cooked fish
 
+	minecraft version = 1.14 or newer
```
Comment (TheBentoBox) [inline foodstuffs.sk:100]: Have you tested that this actually works? Writing the version specifier most of the way down like this? As I understood it a version tag like this specifies how the entire section parses. Have you tested if any of the aliases in this section load in pre-1.14 versions with this change applied?

Comment (ShaneBeee) [reply foodstuffs.sk:100]: Oh dang my apologies, I wasn't aware that is how that works. I'll redo it with a section
EDIT:
Okay I added a section for it.

**Comment by bensku** [comment]
I wouldn't merge this before Spigot 1.15 is out and we have done some testing. Looks good, though!

**Comment by ShaneBeee** [comment]
> I wouldn't merge this before Spigot 1.15 is out and we have done some testing. Looks good, though!

Perfectly fine with me, I wasn't expecting it to get pulled til after, I just wanted to get a head start on it.

**Comment by ShaneBeee** [comment]
Whenever someone wants to get around to it, this can be merged now.
I have tested all the new elements and everything works just great 👍

**Pull request merged**